core/app/views/refinery/admin/_tmp_search.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="search">
+  <form method='GET' action='<%= url %>'>
+    <div class="search-form">
+      <%= text_field :search, nil, :id => 'search', :type => 'search', :name => 'search', :value => local_assigns[:search],
+                     :class => 'form-control',
+                     :placeholder => t('search_input_notice', :scope => 'refinery.admin.search')
+      %>
+      <%= hidden_field :dialog, nil, :name => 'dialog', :id => nil, :value => true if from_dialog? %>
+      <%= hidden_field :callback, nil, :name => 'callback', :id => nil, :value => @callback if @callback.presence %>
+      <span id="refinery-search-btn" class="search-btn">
+        <button type="submit"><i class="icon icon-search"></i>
+          <div class="inline loader xs bump-up"></div>
+        </button>
+      </span>
+    </div><!-- /input-group -->
+  </form>
+</div>
```

Comment:
I'm fine to move this somewhere else.  Wherever it goes, there is going to be a temporary conditional logic to show the current search form html, and the new search form html.  You'd like to see that within refinerycms-search?  I don't see refinerycms-search in my Gemfile.lock.  Would we then be adding this as a new dependency?